hw/ip/kmac/data/kmac.hjson

```diff
@@ -213,6 +213,67 @@
                 Key Derivation Function (KDF).
                 '''
         } // f: sideload
+        { bits: "17:16"
+          name: entropy_mode
+          desc: '''Entropy Mode
+
+          Software selects the entropy source with this field. In EdnMode,
+          the entropy generator sends requests to EDN to get the entropy. The
+          received entropy is fed into internal LFSR.
+
+          In SwMode, the software should update the internal LFSR seed
+          through !!ENTROPY_SEED_LOWER and !!ENTROPY_SEED_UPPER.
+          '''
+
+          enum: [
+            { value: "0"
+              name: "idle_mode"
+              desc: '''At reset state, the entropy mode is Idle mode. It does
+              not operate in this mode.
+              '''
+            }
+            { value: "1"
+              name: "edn_mode"
+              desc: "Entropy generator module fetches entropy from EDN"
+            }
+            { value: "2"
+              name: "sw_mode"
+              desc: '''Software update the internal entropy via register
+              interface'''
+            }
+          ]
+        } // f: entropy_mode
+        { bits: "19"
+          name: entropy_fast_process
+          desc: '''Entropy Fast process mode.
+
+          If 1, entropy logic uses garbage data while not processing the KMAC
```

Comment:
does garbage data basically equal previous entropy in this case?

Reply:
Yes

Reply:
it might be clearer just to say previous entropy directly then.  I guess you do in the sentence afterwards, but i got confused if these were somehow different.